clients/web/src/views/body/AssetstoresView.js

```diff
@@ -61,11 +62,24 @@ var AssetstoresView = View.extend({
         }, this).fetch();
     },
 
+    destroy: function () {
+        this._destroyPlots();
+        View.prototype.destroy.call(this);
+    },
+
+    _destroyPlots: function () {
+        for (let plot of this.plots) {
+            plot.data('jqplot').destroy();
```

Comment:
Without this the views leave detached DOM trees with HTML canvas elements, created by jqplot, at the roots.
https://github.com/jqPlot/jqPlot/blob/460412a84e603a62eea0ecd90057cbaac3e2d9c8/src/jqplot.core.js#L2843